packages/plugins/i18n/admin/src/components/LocalePicker/index.js

```diff
@@ -66,6 +66,7 @@ const LocalePicker = () => {
         page: 1,
         plugins: { ...query.plugins, i18n: { locale: code } },
       });
+      location.reload();
```

Comment:
Why are we reloading? Can you write a comment please.

Reply:
Same as the previous comment but now it's the case when changing the locale and the user is in the "Content type" view